### PR TITLE
Optimized properties table when showing `all` properties

### LINKED ISSUE #331: [Properties] "All properties" needs to be a read only view
The "All Properties" tab is a readonly view. Therefore the inline edit functionallity needs to be disabled and the "Delete" icon should be hidden.

(follow-up to #324)

### PR DESCRIPTION
**Pull request by Corepex**
## Changes in this pull request
Resolves #331 
Blocked by: #461

### CODE REVIEW

**Review by markus-moser — CHANGES_REQUESTED** [review]
@Corepex Thanks, I still found some editable items:

- [ ] drag&drop to asset relations is possible
- [ ] the inheritable checkbox is editable

Could you please check this? Afterwards I'd say it's fine!

**Comment by markus-moser** [comment]
Separate issue for the headline topic #463

File: assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx [hunk 1]
```diff
@@ -125,13 +125,15 @@ export const Table = ({ propertiesTableTab }: ITableProps): React.JSX.Element =>
...
 
-            <Button
-              icon={ <Icon name="trash" /> }
-              onClick={ () => {
-                removeProperty(info.row.original)
-              } }
-              type="link"
-            />
+            {propertiesTableTab === 'own' && (
+              <Button
```
Comment (vin0401) [inline assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx:129]: Now where I see it, we could already implement the new Button with text and icon component here.

**Review by vin0401 — CHANGES_REQUESTED** [review]

File: assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx (hunk 1 above)
Comment (vin0401) [reply assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx:129]: Ahhh, nevermind. :D  

I will refine the api of the button a bit more beforehand. ;D
So far it's not that compatible to ant design.

**Pull request merged**